examples/rvc-app/rvc-common/include/rvc-device.h

```diff
@@ -0,0 +1,77 @@
+#pragma once
+
+#include "rvc-mode-delegates.h"
+#include "rvc-operational-state-delegate.h"
+#include <app/clusters/mode-base-server/mode-base-server.h>
+#include <app/clusters/operational-state-server/operational-state-server.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+
+const EndpointId RvcRunModeEndpoint          = 1;
+const EndpointId RvcCleanModeEndpoint        = 1;
+const EndpointId RvcOperationalStateEndpoint = 1;
+
+class RvcDevice
+{
+private:
+    RvcRunMode::RvcRunModeDelegate mRunModeDelegate;
+    ModeBase::Instance mRunModeInstance;
+
+    RvcCleanMode::RvcCleanModeDelegate mCleanModeDelegate;
+    ModeBase::Instance mCleanModeInstance;
+
+    RvcOperationalState::RvcOperationalStateDelegate mOperationalStateDelegate;
+    OperationalState::Instance mOperationalStateInstance;
+
+public:
+    RvcDevice() :
+        mRunModeDelegate(),
+        mRunModeInstance(&mRunModeDelegate, RvcRunModeEndpoint, RvcRunMode::Id, chip::to_underlying(RvcRunMode::Feature::kOnOff)),
```

Comment:
Re. the last argument (RvcRunMode::Feature::kOnOff) - does that mean that this cluster has a dependency on the OnOff cluster? The related bit in the Feature bit mask should be cleared for this device type.

Reply:
This was a copy paste error. I fixed this [here](https://github.com/project-chip/connectedhomeip/pull/28719/commits/5e2d97b4011955e11754224a1e2ee59cbb4c25a5).